library/net.cpp

```diff
@@ -1569,7 +1569,7 @@ int n2_trans_forward(n2_request_info *request_info, struct dnet_net_state *orig,
 int n2_send_error_response(struct dnet_net_state *st,
                            struct n2_request_info *req_info,
                            int errc,
-                           struct dnet_access_context *context) {
+                           struct dnet_access_context *) {
```

Comment:
Why is `context` omitted? `replier` has to update context with send timings.

Reply:
Ok, I'll put TODO instead of deleting param name.

Reply:
Here I think we should remove `struct dnet_access_context *context` param, because replier will itself embed the context.

Reply:
Fixed: param removed